src/pgroonga.c

```diff
@@ -6365,7 +6365,7 @@ PGrnSearchBuildCondition(Relation index, ScanKey key, PGrnSearchData *data)
 	{
 		grn_obj *keywords = &(buffers->general);
 		grn_obj keywordBuffer;
-		unsigned int i, n;
+		unsigned int i, n, nTargetKeywords = 0;
```

Comment:
`i`と`n`はループを回すための関係する変数だから一緒に宣言してもいいけど、`nTargetKeywords`はそうじゃないから一緒にしないで。

```suggestion
		unsigned int i, n;
		unsigned int nTargetKeywords = 0;
```